src/routes/twofactorauthentication/send-lost-app-request.ts

```diff
@@ -0,0 +1,26 @@
+import Joi from 'joi';
+import { ControllerGeneratorOptionsWithoutClientOrSupplier } from '../../comms/controller';
+
+interface Request {
+  query: {
+    email: string;
+  };
+}
+
+type Response = void;
+
+const controllerGeneratorOptions: ControllerGeneratorOptionsWithoutClientOrSupplier = {
+  method: 'post',
+  path: '/send-lost-app-request',
```

Comment:
Why lost app request? They lost the app? 😆  Is this for forgot password or forgot login or something else?